app/web/features/communities/events/CreateEventPage.tsx

```diff
@@ -27,7 +28,17 @@ const useStyles = makeStyles((theme) => ({
 
 export default function CreateEventPage() {
   const classes = { ...useEventFormStyles(), ...useStyles() };
-  const history = useHistory<{ communityId?: number }>();
+  const router = useRouter();
+
+  const urlCommunityIdString =
+    typeof window !== "undefined"
+      ? stringOrFirstString(router.query["communityId"])
```

Comment:
Are there situations where `router.query` is not consistently returning the same thing? Seems interesting we need to do the extraction ourselves...

Reply:
Ohh so it combines both `3`s into `[3, 3]` if you repeat a URL param but as `3` literally if it only exists once?